instances/latest/terminologies/UBERONParcellation/infundibularOrgan.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.om-i.org/props/"
+  },
+  "@id": "https://openminds.om-i.org/instances/UBERONParcellation/infundibularOrgan",
+  "@type": "https://openminds.om-i.org/types/UBERONParcellation",
+  "definition": "Is a regional part of nervous system. Is part of the central nervous system. [auto-generated from properties of the [UBERON ontology term](http://purl.obolibrary.org/obo/UBERON_0011358) ('is_a' and 'relationship')]",
+  "description": null,
```

Comment:
@UlrikeS91 Should there be a description for this term? Or is the provided "def" not sufficient?